packages/budget-it/src/main.ts

```diff
@@ -2,10 +2,12 @@ import './assets/main.css';
 
 import { createApp } from 'vue';
 import { createPinia } from 'pinia';
+import Tooltip from 'primevue/tooltip';
 import App from './App.vue';
 
 const app = createApp(App);
 
+app.directive('tooltip', Tooltip);
```

Comment:
Ideally, this will come from a directive `@inkbeard/ui-vue`, which will just be an extension of primevue's [tooltip directive](https://primevue.org/tooltip/) or a custom one.